snap/snapcraft.yaml

```diff
@@ -128,10 +128,46 @@ parts:
       - percona-server-mongodb
       - percona-backup-mongodb
       - util-linux
+    override-prime: |
+      craftctl default
+      gzip -df $CRAFT_PRIME/licenses/LICENSE-Community.txt.gz
+      mv $CRAFT_PRIME/licenses/LICENSE-Community.txt \
+        $CRAFT_PRIME/licenses/LICENSE-percona-server
+    organize:
+      usr/share/doc/percona-server-mongodb/LICENSE-Community.txt.gz:
+        licenses/LICENSE-Community.txt.gz
   percona-mongodb-exporter:
     plugin: dump
     source: "https://github.com/percona/mongodb_exporter/releases/download\
       /v0.37.0/mongodb_exporter-0.37.0.linux-64-bit.deb"
   wrapper:
     plugin: dump
     source: snap/local
+  snap-license:
+    plugin: dump
+    source: .
+    source-type: local
+    stage:
+      - licenses/LICENSE-snap
+    organize:
+      LICENSE: licenses/LICENSE-snap
+  percona-mongodb-exporter-license:
+    plugin: nil
+    build-packages:
+      - wget
+    override-build: |
+      PERCONA_GH=https://raw.githubusercontent.com/percona
+      craftctl default
+      mkdir -p $CRAFT_PART_INSTALL/licenses
+      wget ${PERCONA_GH}/mongodb_exporter/v0.37.0/LICENSE \
+        -o $CRAFT_PART_INSTALL/licenses/LICENSE-mongodb-exporter
+  percona-backup-mongodb-license:
+    plugin: nil
+    build-packages:
+      - wget
+    override-build: |
+      PERCONA_GH=https://raw.githubusercontent.com/percona
+      craftctl default
+      mkdir -p $CRAFT_PART_INSTALL/licenses
+      wget ${PERCONA_GH}/percona-backup-mongodb/main/LICENSE \
```

Comment:
Let's use `/usr/share/doc/percona-backup-mongodb/copyright` temporarily for now.
The actual LICENSE file will be included in the deb package during the next pbm release https://github.com/percona/percona-backup-mongodb/pull/814

Reply:
Have you seen the contents of `copyright`?  I don't think it is relevant to the source package whatsoever

Reply:
@jardon actually it is! I learned this week that the `copyright` file is just debian way of licensing. Sorry I missed this when reviewing. It would be best to revert it the mongo-deb part to:
```

      usr/share/doc/percona-server-mongodb/LICENSE-Community.txt.gz: licenses/LICENSE-Community.txt.gz
```

Reply:
I will update this!